syntax = "proto3";

package userservice.v1;

import "google/type/datetime.proto";

message User {
  string id = 1;
  string name = 2;
  string email = 3;
  google.type.DateTime created_at = 4;
}


message GetUserRequest {
  string user_id = 1;
}

message GetUserResponse {
  User user = 1;
}


service UserService {
  rpc GetUser(GetUserRequest) returns (GetUserResponse) {}
}
